doc/memos/rdm0.md

```diff
@@ -0,0 +1,153 @@
+
+- RDM: 0
+- Title: RIOT Developer Memo Format, Publishing and Maintenance Process
+- Author: Emmanuel Baccelli  
+- Date: October 2018
+
```

Comment:
referring (again to Python PEPs) I would add some more header fields such as `type`, e.g. *Informational*, *API*, ..., and `status` e.g. *Draft*, *Active*, *Deprecated*. See https://github.com/python/peps/blob/master/pep-0001.txt

Reply:
also content-type might be helpful